src/gamekit.cs

```diff
@@ -268,14 +276,27 @@ interface GKSession {
 	}
 
 	[Watch (3,0)]
+	[DisableDefaultCtor]
 	[BaseType (typeof (NSObject))]
 	interface GKLeaderboard {
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Use 'LoadEntries (GKLeaderboardPlayerScope, GKLeaderboardTimeScope, NSRange, GKEntriesForPlayerScopeHandler)' instead.")]
+		[Deprecated (PlatformName.TvOS, 14, 0, message: "Use 'LoadEntries (GKLeaderboardPlayerScope, GKLeaderboardTimeScope, NSRange, GKEntriesForPlayerScopeHandler)' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10, 16, message: "Use 'LoadEntries (GKLeaderboardPlayerScope, GKLeaderboardTimeScope, NSRange, GKEntriesForPlayerScopeHandler)' instead.")]
+		[Deprecated (PlatformName.WatchOS, 7, 0, message: "Use 'LoadEntries (GKLeaderboardPlayerScope, GKLeaderboardTimeScope, NSRange, GKEntriesForPlayerScopeHandler)' instead.")]
```

Comment:
are the parameters useful ?

Reply:
good question! There are 2 API that are both named `LoadEntries`, but since these are both new and have different parameter lists I think it is okay to assume that users can figure out which one they need on their own.

Removing parameters :)